R/env.R

```diff
@@ -552,6 +595,26 @@ env_clone <- function(x, parent = env_parent(x)) {
   list2env(as.list(x, all.names = TRUE), parent = parent)
 }
 
+#' Does environment inherits from another environment?
```

Comment:
I was just correcting your English, not making a general statement about grammatical forms :wink: